### ISSUE TITLE
Outdated plugin folders not deleted from Jellyfin's config folder after updates.

### ISSUE THREAD

**Issue by maru801** [nicknsy/jellyscrub#96]
Turns out that after each update of this plugin, Jellyscrub's previous (now old) folder within Jellyfin's config plugin folder is not deleted. They have to be manually deleted instead of the update taking care of it afterwards like other plugins.

**Comment by nicknsy** [comment]
This could be related to the ffmpeg process not being killed when jellyfin stops. Perhaps the the dll is not properly unloaded and is treated as in use. Not sure but will have to look into it.

**Comment by nicknsy** [comment]
Not sure if this is actually specific to jellyscrub. I installed an older version of AniDB and after it was updated through the task, jellyfin didn't auto-remove the folder of the older plugin on shutdown or subsequent startup. Do you have any other specific plugins that you've updated and had them remove the old folder?

**Comment by maru801** [comment]
Unfortunately, I have not checked before during updates. Especially since I have the auto-update task set for all plug-ins, I don't know when they're all updated, and therefore never checked the plugin folder before and after an update.

It could be that leaving the old plugin folders is the default and that some other task cleans up the unused folders eventually. Since I have some plugins that have auto-updated in the past, I would have had left-over folders. But since I first posted this issue, there were no outdated plugin folders apart from Jellyscrub. That is what led me to believe that Jellyscrub was at fault.

Maybe you're right. I just came across this when I was moving my config folders to a new location and noticed that the old plugin folder was remaining for Jellyscrub.
